pint/phase.py

```diff
@@ -49,6 +50,7 @@ def __new__(cls, arg1, arg2=None):
             return super(Phase, cls).__new__(cls, ii.to(u.cycle), ff.to(u.cycle))
 
     def __neg__(self):
+        #TODO: add type check for __neg__ and __add__
```

Comment:
No phase wrap issue (I don't think), Scott and I were worried there's nothing raising an exception if Phase is being given something that can't become a phase (string, list, etc.) rather than something that will work (int, tuple, float)